datumaro/plugins/voc_format/format.py

```diff
@@ -171,6 +200,51 @@ def write_label_map(path, label_map):
 
             f.write('%s\n' % ':'.join([label_name, color_rgb, parts, actions]))
 
+def write_meta_file(path, label_map):
+    dataset_meta = {}
+
+    labels = {}
+    segmentation_colors = []
+    parts = {}
+    actions = {}
+
+    i = 0
+    for label_name, label_desc in label_map.items():
+        labels[str(i)] = label_name
+
+        segmentation_colors.append([int(label_desc[0][0]), int(label_desc[0][1]), int(label_desc[0][2])]
+            if label_desc[0] != None else None)
+
+        if not isinstance(label_desc, tuple) and \
+                label_desc[1] is not None and 0 < len(label_desc[1]):
+            parts[str(i)] = label_desc[1]
+
+        if not isinstance(label_desc, tuple) and \
+                label_desc[2] is not None and 0 < len(label_desc[2]):
+            actions[str(i)] = label_desc[2]
+        i += 1
+
+    dataset_meta['label_map'] = labels
+
+    if any(segmentation_colors):
+        bg_label = find(label_map.items(), lambda x: x[1] == (0, 0, 0))
+        if bg_label is not None:
+            dataset_meta['background_label'] = str(bg_label[0])
+
+        dataset_meta['segmentation_colors'] = segmentation_colors
```

Comment:
Does AC support `None` as a label color?

Reply:
No, it's probably going to be a problem.
But Datumaro is able to work with `None`, I would not like to replace it with something when saving meta file.

Reply:
Should we align with them on this topic? Datumaro is not the main consumer for these files, I suppose.

Reply:
Fixed this problem.